### ISSUE TITLE
Why is scitos_apps in the strands_morse and strands_scitos rosintall files?

### ISSUE THREAD

**Issue by Jailander** [strands-project/strands_systems#45]
Shouldn't it be in strands_base instead?

**Comment by cdondrup** [comment]
Because it is not needed if you do not have the robot or the simulator present. So no need for it in the base system, imho.

**Comment by Jailander** [comment]
good point, I was just wondering why would you use the system without the robot or the simulator and in such a case why couldn't you use things that are scitos_apps like scitos_cmd_vel_mux
